src/bin/pg_autoctl/cli_enable_disable.c

```diff
@@ -126,6 +129,39 @@ cli_secondary_getopts(int argc, char **argv)
 				break;
 			}
 
+			case 'V':
```

Comment:
shall we consider `keeper_cli_formation_create_getopts` too

Reply:
Done now!